README.md

```diff
@@ -137,11 +137,20 @@ module.exports.maintainFavoritesCount = integrify({
   rule: 'MAINTAIN_COUNT',
   source: {
     collection: 'favorites',
-    foreignKey: 'articleId',
   },
   target: {
-    collection: 'articles',
+    collection: 'articles/$source.articleId', // NOTE: This collection needs to reference a document
+    // OR
+    collection: 'articles/$source.fieldValue/private/details', // Can reference a field value (requires source), will throw error if it doesn't exist
     attribute: 'favoritesCount',
```

Comment:
It's called collection but is actually a reference to a document. Should the name be document or collection?

Reply:
tbh im still finding the documentation super confusing, its not clear to me what the structure of the db / should be, maybe we should add a table showing the db state before and after?